skbio/stats/composition.py

```diff
@@ -1105,6 +1212,34 @@ def _gram_schmidt_basis(n):
     return basis.T
 
 
+def _build_basis(sbp):
+    """
+    Builds an orthogonal basis from a sequential binary partition. 
+       
+    "Code for a SBP. The first row represents the labels of parts of the compositional vector. At each order of
+    partition, +1 means that the part is assigned to the first group, −1 to the second group, and 0 that this part is not
+    in the group which is divided at this order."
+    J.J. Egozcue and V. Pawlowsky-Glahn, http://dugi-doc.udg.edu/bitstream/handle/10256/660/Egozcue-Pawlowsky.pdf
```

Comment:
I realized that we didn't have the same definition for a "basis". I corrected the output of the `_build_base` method.